1249/en.md

```diff
@@ -0,0 +1,95 @@
+# LOJ 1249 - Chocolate Thief
+
+### Problem statement
+In this problem, each test case contains `n` number of lines. Each line contains:
+ `Name(n) length(l),breadth(b) height(h)`
+
+Denoting that the person with name(n) ate the cake of the length(l),breadth(b) & height. Each information separated by a `whitespace` .Each person should eat the same amount of cake `in volume` i.e each `l*b*h` should be the same.
+However, one single `thief` stole a portion from another `victim`, given that in all cases, 
+```
+number of thief = number of victim = 1
+```
+We need to print out the `name` of the `thief` & `victim` in a particular testcase.
+
+**Example Output:** `Case 1: mun took chocolate from all` 
+
+### Approach
+
+Basically we need to find out the name associated with `higher than average volume`  and `lower than average volume` 
+of consumed cake. I used this data-structure to solve the problem:
```

Comment:
Please explain the structure a bit, basically, you are using a map to track name and frequency. I can see you added a comment on each line, but just write a paragraph saying the same. Also, its good to mention what this structure is as many people do not have an understanding of Javascript.